lib/ansible/modules/network/avi/avi_serviceenginegroup.py

```diff
@@ -100,23 +106,26 @@
             - Field introduced in 18.2.3.
             - Default value when not specified in API or module is interpreted by Avi Controller as 0.
         version_added: "2.9"
+        type: int
     archive_shm_limit:
         description:
             - Amount of se memory in gb until which shared memory is collected in core archive.
             - Field introduced in 17.1.3.
             - Default value when not specified in API or module is interpreted by Avi Controller as 8.
+        type: int
     async_ssl:
         description:
-            - Ssl handshakes will be handled by dedicated ssl threads.
+            - Ssl handshakes will be handled by dedicated ssl threads.requires se reboot.
```

Comment:
```suggestion
            - Ssl handshakes will be handled by dedicated ssl threads.requires se reboot.
```
This is not correct English.